src/main/java/com/yegor256/tojos/Csv.java

```diff
@@ -89,7 +91,15 @@ public Csv(final Path path) {
     public Collection<Map<String, String>> read() {
         final Collection<Map<String, String>> rows = new LinkedList<>();
         if (Files.exists(this.file)) {
-            try (CSVReader reader = new CSVReader(Files.newBufferedReader(this.file))) {
+            // @checkstyle MethodBodyCommentsCheck (1 line)
```

Comment:
@baudoliver7 just move the comment to the javadoc block of the method

Reply:
Ok